h/static/scripts/directive/selection-tabs.js

```diff
@@ -1,23 +1,27 @@
 'use strict';
 
+var uiConstants = require('../ui-constants');
+
 module.exports = function () {
   return {
     bindToController: true,
     controllerAs: 'vm',
     //@ngInject
     controller: function (annotationUI) {
+      this.tabAnnotations = uiConstants.TAB_ANNOTATIONS;
```

Comment:
I would suggest keeping the property naming `TAB_ANNOTATIONS` to make it obvious that this is a template. The other approach, which I think is slightly preferable, is to extract the logic out of the template and into a helper function on the controller. That makes it easier to test and avoids creating long unreadable lines in the template.